### DISCUSSION TITLE
Tutoría.

### DISCUSSION THREAD

**Discussion by JavierLianoRioz** [mmasias/24-25-PRG1#578] [General]
Mañana, Jueves 31, nos interesaría tener una tutoría de programación de 10:15 a 12:15.

**Comment by mmasias** [comment]
Anotada: de 10 a 12, en la -2.6

**Reply by mmasias** [reply]
@JavierLianoRioz: perdonar! Mañana estaré fuera a esa hora 😞 ¿podría ser a las 14h, después de su clase de iGPySw?
